### ISSUE TITLE
Symfony dependency injection install error

### ISSUE THREAD

**Issue by icringasu-ro** [moxiworks-platform/moxiworks-php#9]
Please update version for symfony/dependency-injection.  Required version ~3.4 is too old and it is not compatible with Laravel 7+

**Comment by zone44** [comment]
Yes please, I get the following errors when trying to install using composer:
    - moxiworks/moxiworks_platform 0.13.17 requires symfony/dependency-injection ~3.4 -> satisfiable by symfony/dependency-injection[v3.4.0, ..., v3.4.47].
    - symfony/event-dispatcher v5.2.4 conflicts with symfony/dependency-injection v3.4.47.
    - symfony/event-dispatcher v5.2.4 conflicts with symfony/dependency-injection v3.4.46.
    - symfony/event-dispatcher v5.2.4 conflicts with symfony/dependency-injection v3.4.45.
    - symfony/event-dispatcher v5.2.4 conflicts with symfony/dependency-injection v3.4.44.
    - symfony/event-dispatcher v5.2.4 conflicts with symfony/dependency-injection v3.4.43.
    - symfony/event-dispatcher v5.2.4 conflicts with symfony/dependency-injection v3.4.42.
    - symfony/event-dispatcher v5.2.4 conflicts with symfony/dependency-injection v3.4.41.
    - symfony/event-dispatcher v5.2.4 conflicts with symfony/dependency-injection v3.4.40.
    - symfony/event-dispatcher v5.2.4 conflicts with symfony/dependency-injection v3.4.39.
    - symfony/event-dispatcher v5.2.4 conflicts with symfony/dependency-injection v3.4.38.
    - symfony/event-dispatcher v5.2.4 conflicts with symfony/dependency-injection v3.4.37.
    - .......................
    -
